.github/workflows/build-and-publish-chart.yml

```diff
@@ -50,18 +50,24 @@ jobs:
         run: |
           helm dependency update charts/cloudzero-agent/
 
-      - name: Update Version
-        id: version
-        uses: flatherskevin/semver-action@v1
+      - name: 'Get Previous tag'
+        id: previoustag
+        uses: "WyriHaximus/github-action-get-previous-tag@v1"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: 'Get next minor version'
```

Comment:
what happens when we want to do a minor or major release? does this still get the next patch version?

Reply:
This was hard coded as path before. I think we should adjust this workflow over a few more PRs:
1. Add ability to specify a version as `input`. so we can force the version setting...
2. Ability to generate the change log, with instructions on installation at done here: https://github.com/Cloudzero/cloudzero-agent-validator/blob/develop/.github/workflows/release-to-main.yml#L65-L94

That said, I wanted to stick to what was done, and split the changes over a few PRs. Is that reasonable?